src/w3c/informative.js

```diff
@@ -1,10 +1,13 @@
 // Module w3c/informative
 // Mark specific sections as informative, based on CSS
+import "deps/hyperhtml";
 export const name = "w3c/informative";
 
-export function run(conf, doc, cb) {
-  $("section.informative")
-    .find("> h2:first, > h3:first, > h4:first, > h5:first, > h6:first")
-    .after("<p><em>This section is non-normative.</em></p>");
-  cb();
+export function run() {
+  Array.from(document.querySelectorAll("section.informative"))
+    .map(informative => Array.from(informative.children).find(child => child instanceof HTMLHeadingElement))
+    .filter(heading => heading)
+    .forEach(heading => {
+      heading.parentNode.insertBefore(hyperHTML`<p><em>This section is non-normative.</em></p>`, heading.nextSibling);
```

Comment:
Nit: this should be a reduce.

Reply:
I'm not very familiar with `.reduce`, how should the change look like?

Reply:
Actually, you are correct here. I misread the code. No need for reduce.